student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown', parent_permission: true)
+    @classroom = classroom
+    super(age, name, parent_permission: parent_permission)
+  end
```

Comment:
- [ Required ] Please consider making the call to the `superclass` constructor the first statement in your `base class`, this is usually the norm in most Object Oriented programming languages.

```RUBY
    def initialize(classroom, age, name = 'unknown', parent_permission: true)
       super(age, name, parent_permission)
       @classroom = classroom  
    end
```

Reply:
Similar to the change that you required above, I have also made the call to the superclass constructor the first statement In my base class. Thank you for the helpful insight offered. 👏🏼